test/Atc.Rest.ApiGenerator.Tests/SyntaxGenerators/Api/SyntaxGeneratorTestBase.cs

```diff
@@ -43,27 +43,36 @@ protected async Task ExecuteTest(YamlSpecFile specFile)
             await VerifyGeneratedCode(generatedCode, verifySettings);
         }
 
-        private VerifySettings CreateVerifySettings(YamlSpecFile yamlFile, ApiProjectOptions apiOptions)
+        private VerifySettings CreateVerifySettings(GeneratorTestInput yamlFile, ApiProjectOptions apiOptions)
         {
             var settings = new VerifySettings();
-            settings.UseDirectory(yamlFile.DirectoryName);
-            settings.UseFileName(yamlFile.FileName);
+            settings.UseDirectory(yamlFile.TestDirectory);
+            settings.UseFileName(yamlFile.TestName);
             settings.UseExtension("cs");
             settings.AddScrubber(input => input.Replace(apiOptions.ToolVersion.ToString(), "x.x.x.x"));
             return settings;
         }
 
-        private static IReadOnlyList<YamlSpecFile> GetYamlFiles([CallerFilePath] string sourceFilePath = "")
+        private static IReadOnlyList<GeneratorTestInput> CreateTestInput([CallerFilePath] string sourceFilePath = "")
```

Comment:
Is this a better name?
```suggestion
        private static IReadOnlyList<GeneratorTestInput> GetTestInput([CallerFilePath] string sourceFilePath = "")
```

Reply:
it is, thanks for the suggestion.